### PR TITLE
feat: add bulk negative scenario tests for invalid and inactive fspId

### PR DESCRIPTION
**Pull request by kleyow**
image

### CODE REVIEW

File: collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json [hunk 1]
```diff
@@ -0,0 +1,384 @@
+{
+  "name": "multi",
+  "test_cases": [
+    {
+      "id": 1,
+      "name": "negative scenario - invalid payee name in request",
```
Comment (elnyry-sam-k) [inline collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json:6]:
```suggestion
      "name": "negative scenario - invalid Payee FSP ID in request",
```

File: collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+      "meta": {
+        "info": "negative scenario - invalid payee name"
```
Comment (elnyry-sam-k) [inline collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json:8]:
```suggestion
        "info": "negative scenario - invalid Payee FSP ID"
```

File: collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+                "}"
+              ]
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": 2,
+      "name": "negative scenario - invalid payer name in request",
```
Comment (elnyry-sam-k) [inline collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json:255]:
```suggestion
      "name": "negative scenario - invalid Payer FSP ID in request",
```

**Comment by elnyry-sam-k** [comment]
Thanks Kevin, good to have this PR..

In general, we need to use Payer FSP / Payee FSP instead of Payer and Payee to avoid any ambiguity..

**Comment by kleyow** [comment]
@elnyry-sam-k done.

**Comment by kleyow** [comment]
```
Test Suite:GP Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 2573                          │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 2573                          │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 587                           │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 129                           │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Fri, 29 Jul 2022 14:41:14 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ Fri, 29 Jul 2022 14:44:15 GMT │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 181156 ms                     │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-multi-2022-07-29T14:44:15.864Z.html was generated
Terminate with exit code 0
Done

Test Suite:Bulk Tests
Environment:Development
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 183                           │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 183                           │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 17                            │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 4                             │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Fri, 29 Jul 2022 14:44:44 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ Fri, 29 Jul 2022 14:45:08 GMT │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 23964 ms                      │
└───────────────────┴───────────────────────────────┘
TTK-Assertion-Report-multi-2022-07-29T14:45:08.641Z.html was generated
Terminate with exit code 0
Done


```

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**